aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -412,6 +412,16 @@ struct Vec256 {
   Vec256<T> reciprocal() const {
     return map([](T x) { return (T)(1) / x; });
   }
+  Vec256<T> rad2deg(const double M_180_PI) const {
+    Vec256<T> ret;
+    for (int64_t i = 0; i != size(); i++) {
+      ret[i] = (T)(M_180_PI) * values[i];
+    }
+    return ret;
+  }
+  Vec256<T> deg2rad() const {
+    return map([](T x) { return (T)(M_PI) * x / (T)(180.0); });
```

Comment:
Why is this one not using a precomputed value? For numerical precision?

Reply:
I just implemented rad2deg first to take feedback and improve it. I'll incorporate that feedback in both rad2deg and deg2rad

Reply:
Sounds good!